velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -699,5 +699,26 @@ TEST_F(ArithmeticTest, truncate) {
   EXPECT_DOUBLE_EQ(truncate(123456789012345678901.23, -21).value(), 0.0);
 }
 
+TEST_F(ArithmeticTest, BetaCDF) {
+  const auto beta_cdf = [&](std::optional<double> a,
+                            std::optional<double> b,
+                            std::optional<double> value) {
+    return evaluateOnce<double>("beta_cdf(c0, c1, c2)", a, b, value);
+  };
+
+  EXPECT_EQ(0.09888000000000001, beta_cdf(3, 4, 0.2));
```

Comment:
Please add more test cases especially around boundary values, and special values like std::numeric_limits<T>::infinity(),
std::numeric_limits<T>::max()/min(), std::numeric_limits<T>:quiet_NaN()

Reply:
Added testcases.